docs/how-to/installation/aws.txt

```diff
@@ -46,148 +45,134 @@ Installation
 Deploying
 ~~~~~~~~~
 
+The basic command to deploy a cluster is as follows:
+
 .. code::
 
    det-deploy aws up --cluster-id CLUSTER_ID --keypair KEYPAIR
 
 ``CLUSTER_ID`` is an arbitrary unique ID for the new cluster.
 We recommend choosing a cluster ID that is memorable and
 helps identify what the cluster is being used for. The cluster ID will be
-used as the AWS CloudFormation Stack Name.
+used as the AWS CloudFormation stack name.
 
-``KEYPAIR`` is the name of the AWS EC2 Keypair to use when provisioning
+``KEYPAIR`` is the name of the AWS EC2 key pair to use when provisioning
 the cluster. If the AWS CLI is installed on your machine, you can get a
 list of the available keypair names by running ``aws ec2 describe-key-pairs``.
 
-The deployment process may take 5-10 minutes. When it completes, summary information about the newly deployed cluster will be printed, including the URL of the Determined master.
+The deployment process may take 5--10 minutes. When it completes, summary information about the newly deployed cluster will be printed, including the URL of the Determined master.
 
 .. _determined-deploy-deployment-types:
 
 Deployment Types
 ~~~~~~~~~~~~~~~~
 
-Simple
-^^^^^^
-The simple deployment provides an easy way to deploy a Determined cluster in AWS.
-This creates the master instance in the default subnet for the account.
-
-VPC
-^^^
-The VPC deployment creates a separate VPC with public subnets, and the Determined
-cluster is deployed into these subnets.
-
-Secure
-^^^^^^
-The secure deployment creates resources to lock down the Determined cluster. These resources are:
-
-- A VPC with a public and private subnet
+``determined-deploy`` supports multiple different deployment types to
+work with different security needs. The deployment type can be specified
+using, e.g., ``det-deploy aws --deployment-type secure``.
```

Comment:
super duper non-blocking: "using the ``--deployment-type`` argument, for example, ..."